### PR TITLE
feat(config): adding Technisat shutter-switch.

### PR DESCRIPTION
**Pull request by cefinger**
Added these devices because I needed them. The configuration works for my devices. The files are adapted copies of existing configuration files. Running 'lerna run lint:config' produces same warnings for these files like for the originals.

### CODE REVIEW

**Comment by Technikte** [comment]
@blhoward2 It get quite a bit messy with device files now, 2 Pull requests kinda redundant ..any Idea about it ?
Already existing Technisat Switch Pull Request https://github.com/zwave-js/node-zwave-js/pull/1603

**Comment by cefinger** [comment]
Sorry didn't see Pull Request #1603. So can remove one of the files in this request. The other configuration file is for 0x0299-0x1a93-0x0005.

**Comment by AlCalzone** [comment]
Please cross-check with https://github.com/zwave-js/node-zwave-js/pull/1603/files - there were some changes to the parameters labels there which also apply here.

File: packages/config/config/devices/0x0299/1a93.json [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+			"unit": "minutes",
+			"minValue": 0,
+			"maxValue": 30240,
+			"defaultValue": 60,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"4": {
+			"label": "Forced Roller Shutter calibration",
```
Comment (cefinger) [inline packages/config/config/devices/0x0299/1a93.json:65]: labels taken from "Fibargroup FGR222"

File: packages/config/config/devices/0x0299/1a93.json [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Deactivated",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0299/1a93.json]:
```suggestion
					"label": "Idle",
```

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**